src/pages/Clients/Execution/Reth.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+import rethBg from '../../../static/reth-bg.png';
+import {
+  Hero,
+  SectionTitle,
+  ValidatorClientPageStyles,
+} from '../ValidatorClientComponents';
+import { PageTemplate } from '../../../components/PageTemplate';
+import { Text } from '../../../components/Text';
+import { Link } from '../../../components/Link';
+
+// eslint-disable-next-line no-unused-vars
+export const RethDetails = () => (
+  <>
+    <SectionTitle level={2} className="mb5">
+      Reth
+    </SectionTitle>
+    <Text className="mt10">
+      <FormattedMessage defaultMessage="Reth is a modular, contributor-friendly and blazing-fast implementation of the Ethereum protocol, in Rust. Reth is able to sync an archive node in less than 50 hours from genesis. A Reth mainnet archive node is only 2.1TB." />
```

Comment:
These numbers are time-dependent, and we tend to avoid updating the docs over time. What do you think about either changing the description or adding the timestamp? e.g.,
```suggestion
      <FormattedMessage defaultMessage="Reth is a modular, contributor-friendly and blazing-fast implementation of the Ethereum protocol, in Rust. Reth is able to sync an archive node in less than 50 hours from genesis. A Reth mainnet archive node is only 2.1TB in June 2024." />
```